packages/waku/src/lib/plugins/vite-plugin-fs-router-typegen.ts

```diff
@@ -225,7 +225,6 @@ export const fsRouterTypegenPlugin = (opts: { srcDir: string }): Plugin => {
         await writeFile(outputFile, formatted, 'utf-8');
       };
 
-      server.watcher.add(opts.srcDir);
```

Comment:
Removing this fixes the issue of failing on windows and ubuntu. @tylersayshi

Reply:
What if we did `server.watcher.add(pagesDir)` instead? .gen.ts will be outside the pages dir so this should be fine I'd think

Reply:
We can try, but is removing this a problem? It still generates .gen.ts file.

Reply:
The idea for adding it here is that it will updates pages.gen.ts when new pages are added or removed. Without the watcher I don't think it will be able to update while the dev server is running. Just on startup only.

Reply:
works on mac... I'll watch to see if the other tests pass but I expect them to 🤞 

https://github.com/dai-shi/waku/pull/1023

Reply:
looks like it passed 🎉